packages/tools/src/utilities/index.ts

```diff
@@ -5,7 +5,6 @@ import {
 
 // Lodash/common JS functionality
 import debounce from './debounce';
-import deepMerge from './deepMerge';
 import throttle from './throttle';
```

Comment:
so you are moving the deepMerge into the core (which is fine), but removing it here, but not removing it from the src/utilities/deepMerge? I'm confusied

Reply:
I think this was actually from your branch, I can remove the actual file too

Reply:
actually i'm not sure what you mean by the above, the `src/utilities/deepMerge` is indeed moved?